cantabular/dimensions.go

```diff
@@ -148,3 +148,29 @@ func (c *Client) GetDimensionOptions(ctx context.Context, req GetDimensionOption
 
 	return &resp.Data, nil
 }
+
+func (c *Client) GetAreas(ctx context.Context, req QueryData) (*GetAreasResponse, error) {
+	resp := &struct {
+		Data   GetAreasResponse `json:"data"`
+		Errors []gql.Error      `json:"errors,omitempty"`
+	}{}
+
+	data := QueryData{
+		Dataset: req.Dataset,
+		Text:    req.Text,
+	}
+
```

Comment:
Why do we need to instantiate `data` since you already have the data on `req`?

You could pass it to the private function, since the `GetAreas(ctx context.Context, req QueryData)` is already documenting that it is of type `QueryData`
```
if err := c.queryUnmarshal(ctx, QueryAreasByArea, req, resp); err !=
```

Reply:
I think that's sensible ^

Reply:
This has now been updated